concent_api/utils/logging.py

```diff
@@ -117,12 +129,13 @@ def log_message_not_allowed(endpoint: str, client_public_key: bytes, method: str
 
 
 def log_subtask_stored(
-    task_id:              str,
-    subtask_id:           str,
-    state:                str,
-    provider_public_key:  bytes,
+    logger,
+    task_id: str,
+    subtask_id: str,
+    state: str,
+    provider_public_key: bytes,
     requestor_public_key: bytes,
-    next_deadline:        int          = None,
+    next_deadline: int = None,
```

Comment:
please change to `Optional[int]` or remove ` = None` if this parameter is mandatory

Reply:
done